presqt/api_v1/views/target.py

```diff
@@ -57,6 +55,7 @@ class TargetDetails(APIView):
 
     def get(self, request, target_name):
         """
+        Retrieve details about a specific Target.
```

Comment:
So, I know that the DRF documentation uses List/Details in the class names.
In my opinion, it's better to tie into the common parlance of collection vs. resource that is integral to the RESTful paradigm.

So class `TargetCollection` and (`Target` or even `TargetResource`). 

Just an opinion, you're the one doing it, so you decide!